src/main/java/org/jitsi/videobridge/octo/OctoEndpoint.java

```diff
@@ -81,7 +81,7 @@ public void requestKeyframe(long mediaSsrc)
     @Override
     public boolean shouldExpire()
     {
-        return ArrayUtils.isNullOrEmpty(getMediaStreamTracks());
```

Comment:
@bbaldino this line is the actual fix. It takes into account all signaled SSRCs (which are now removed), instead of using the tracks (which are video only).

Reply:
ah gotcha, good to know.  what is the other change for?

Reply:
For setting the SSRCs globally so that they are cleaned up properly.